### ISSUE TITLE
Loss and mAP

### ISSUE THREAD

**Issue by JeasonUESTC** [eriklindernoren/PyTorch-YOLOv3#387]
When using this version of the code for training,  a loss not convergence, the mAP on the validation set has reached the maximum value of the whole process of training, mAP instead of reduce a lot when loss of convergence . Using different data sets to train is the same problem. Is somebody in the same situation?You the problem of code?

**Comment by Flova** [comment]
This part of the code has been updated or removed. I, therefore, close this issue now. Feel free to reopen for further discussion.
